### PR TITLE
Added to plot plugin y max and min values

### PR DESCRIPTION
**Pull request by ahcorde**
To improve the visualization of the plots I have include two new field to adjust the y axis

 - y_axis_minimum: Minimum y-axis value
 - y_axis_maximum: Maximum y-axis value

### CODE REVIEW

File: ros_buildfarm/config/plot_config.py [hunk 1]
```diff
@@ -42,6 +42,9 @@ def __init__(self, name, data):
             'stackedBar', 'stackedBar3d', 'waterfall')
 
         self.y_axis_exclude_zero = bool(data.get('y_axis_exclude_zero', False))
+
```
Comment (dirk-thomas) [inline ros_buildfarm/config/plot_config.py]: Please address the CI failure: https://travis-ci.org/ros-infrastructure/ros_buildfarm/jobs/627639835

Comment (ahcorde) [reply ros_buildfarm/config/plot_config.py]: Done

**Comment by cottsay** [comment]
Won't this make it impossible to auto-scale the graphs?

If the intent here is to address the messages we're seeing in the Jenkins logs, maybe we should just fix the bug in the plugin instead of losing autoscale entirely.

For an intermediate fix, this PR could be modified to not default the min and max values. We could then set static values in our build configs, and remove them once auto-scaling is fixed in the plugin. That way we wouldn't need to modify `ros_buildfarm` again.

Thoughts?

**Comment by dirk-thomas** [comment]
> this PR could be modified to not default the min and max values. We could then set static values in our build configs, and remove them once auto-scaling is fixed in the plugin.

:+1:

**Comment by ahcorde** [comment]
I created this PR to improve the plots' visualization. Sometimes the axis are not consistent between different plots. 

For example, the CPU usage makes sense to have values between 0 to 100. Using autoscale, probably, you will have values between 5 to 30 which potencially can create confusion. 

Also other measurements are bounded in some specific values. For example:
 - Round trip 0 - 100 ms
 - Virtual Memory 0 - 1000Mb
 - Resident anonimous memory 0 - 100 Mb
 - Physical memory 0 - 100 Mb

If some of the values are outside the limit probably are an indicator that you should have a look to a specific DDS implementation or rmw.

Maybe if you need to review these plots repetitively, having a fixed y axis will help you to review it faster.

**Comment by dirk-thomas** [comment]
All agreed that we want to use fixed intervals and should specify them in config files. If we don't specify any it would be good to fallback to auto scaling though.

**Comment by ahcorde** [comment]
sorry i misunderstood, with these new changes should be enough

File: ros_buildfarm/config/plot_config.py [hunk 2]
```diff
@@ -43,6 +43,15 @@ def __init__(self, name, data):
 
         self.y_axis_exclude_zero = bool(data.get('y_axis_exclude_zero', False))
 
+        self.y_axis_minimum_exists = 'y_axis_minimum' in data.keys()
+        self.y_axis_maximum_exists = 'y_axis_maximum' in data.keys()
```
Comment (dirk-thomas) [inline ros_buildfarm/config/plot_config.py]: There is no need for a second set of variables. The state that no minimum/maximum exists can be expressed by setting the limits to `None`.

Comment (ahcorde) [reply ros_buildfarm/config/plot_config.py]: Removed bool variables

**Pull request merged**